src/components/googleLogin/googleLogin.js

```diff
@@ -0,0 +1,29 @@
+import { GoogleLogin } from 'react-google-login';
+import React from 'react';
+// eslint-disable-next-line sort-imports-es6-autofix/sort-imports-es6
+import { clientId } from '../../constants/constants';
```

Comment:
Please use appropriate names, Like constants/typeOfConstants ... It will be confusing for sure constants/constants ..